### DISCUSSION TITLE
Where is the option for resolution?

### DISCUSSION THREAD

**Discussion by M0REKZ** [Wohlstand/TheXTech#387] [Q&A] [answered]
I remember there was an option to change resolution, what happened?
<img width="794" alt="Captura de Pantalla 2022-01-05 a la(s) 18 46 36" src="https://user-images.githubusercontent.com/60852359/148294356-e03a9305-89f2-4157-82d0-7d76244dd484.png">

**Comment by Wohlstand** [comment, marked as answer]
It's at `devel` branch, will come at `master` (which is used for mainstream releases) as soon as #232 will be completed.

**Comment by M0REKZ** [comment]
ah ok, thanks
